### DISCUSSION TITLE
[Rating] heart displays a star

### DISCUSSION THREAD

**Discussion by hakeem86** [fomantic/Fomantic-UI#401] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Bug Report

In version 2.6.4 my fiddle would display a heart, in latest version it displays a star.

## Expected result
A heart icon is displayed

## Actual result
A star icon is displayed

## Testcase
https://jsfiddle.net/58armz9s/

## Version
2.7.1

**Comment by lubber-de** [comment]
The Rating module was improved in 2.7, so you can use any icon you want, but needs a change in your code
https://fomantic-ui.com/modules/rating.html#icon

Star is the default, you have to add `data-icon="heart"` now
I adjusted your fiddle accordingly here https://jsfiddle.net/ouw6n9cs/

**Comment by lubber-de** [comment, marked as answer]
You also need to add the color you want, so if the heart should be displayed in red as before , use `ui red rating`...but now you can use any color you want from the fui color palette :slightly_smiling_face: 
Adjusted fiddle: https://jsfiddle.net/ouw6n9cs/1/

**Comment by hakeem86** [comment]
Sorry my mistake for not checking the updated documentation
